code_challenges.rb

```diff
@@ -20,15 +25,67 @@
 album3 = 'Abbey Road'
 # Expected output: 'bby Rd'
 
+# make a method that takes a str parameter
+def vowless(str)
+    str.delete!('aeiouAEIOU') #remove all vowels using delete with vowels as the argument lower and uppercase / .downcase is also an option but less accurate to expected outputs
+end
+
+p vowless album1
+p vowless album2
+p vowless album3
+p album1  #adding bang operator mutates the actual string, without it, you only get the immediate change when the function is run
+p album2
+p album3
+
 
 
 # --------------------3a) Create a class called Bike that is initialized with a model, wheels, and current_speed. The default number of wheels is 2. The current_speed should start at 0. Create a get_info method that returns a sentence with all the data from the bike object.
 
 # Expected output example: 'The Trek bike has 2 wheels and is going 0 mph.'
 
 
+#create a bike class
+class Bike
+attr_accessor :model, :wheels, :current_speed #use attr_accessor for default getter and setter methods
+    def initialize model, wheels, current_speed #initialize the attributes model, wheels, and current_speed
+        @model = model #leave model open to an argument
+        @wheels = 2 #set wheels to default 2
```

Comment:
Remember if we set the @variables to something other than the parameter we don't need the param